en/src/type-conversions/from-into.md

```diff
@@ -36,7 +36,8 @@ fn main() {
     assert_eq!(i1, 0);
 
     // FIX the error in two ways
-    // 1. impl From<char> for ? , maybe you should check the docs mentiond above to find the answer
+    /* 1. impl From<char> for ? , maybe you should check the docs mentiond above 
```

Comment:
not pretty here :P

Reply:
wdym? are you concerned with the list, I don't get it yet =(